src/Uno.UI/Extensions/ViewExtensions.wasm.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+using ViewGroup = Windows.UI.Xaml.UIElement;
+using View = Windows.UI.Xaml.UIElement;
+
+namespace Uno.UI
+{
+	public static partial class ViewExtensions
+	{
+		/// <summary>
+		/// Find the first child of a specific type.
+		/// </summary>
+		/// <typeparam name="T">Expected type of the searched child</typeparam>
+		/// <param name="view"></param>
+		/// <param name="childLevelLimit">Defines the max depth, null if not limit (Should never be used)</param>
+		/// <param name="includeCurrent">Indicates if the current view should also be tested or not.</param>
+		/// <returns></returns>
+		public static T FindFirstChild<T>(this ViewGroup view, int? childLevelLimit = null, bool includeCurrent = true)
+			where T : View
+		{
+			return view.FindFirstChild<T>(null, childLevelLimit, includeCurrent);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'view' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)